app/src/main/res/values/strings.xml

```diff
@@ -60,123 +60,123 @@
-    protected
+    <string name="status_protected" description="This is a large message in a reassuring color. It indicates DNS queries can't be surveilled or modified in transit.">
+    Protected.
     </string>
 
-    <string name="status_app_error">app error</string>
+    <string name="status_app_error">App error.</string>
 
-    <string name="status_dns_error">dns error</string>
+    <string name="status_dns_error">DNS error.</string>
 
-    <string name="status_dns_server_down">dns server down</string>
+    <string name="status_dns_server_down">DNS server down.</string>
 
-    <string name="status_no_internet">no internet</string>
+    <string name="status_no_internet">Not connected to the Internet.</string>
 
-    <string name="status_protected_with_tor">protected with tor</string>
+    <string name="status_protected_with_tor">Protected with Tor</string>
 
-    <string name="status_protected_with_private_dns">protected with private dns</string>
+    <string name="status_protected_with_private_dns">Protected with private DNS.</string>
 
-    <string name="status_protected_with_tor_private_dns">protected with tor &amp; private dns</string>
+    <string name="status_protected_with_tor_private_dns">Protected with Tor and private DNS.</string>
 
-    <string name="status_protected_with_proxy_private_dns">protected with proxy &amp; private dns</string>
+    <string name="status_protected_with_proxy_private_dns">protected with proxy and private DNS.</string>
 
-    <string name="status_protected_with_socks5_private_dns">protected with socks5 &amp; private dns</string>
+    <string name="status_protected_with_socks5_private_dns">Protected with SOCKS5 and private DNS.</string>
 
-    <string name="status_protected_with_http_private_dns">protected with http proxy &amp; private dns</string>
+    <string name="status_protected_with_http_private_dns">Protected with http proxy and private DNS.</string>
 
-    <string name="status_protected_with_socks5">protected with socks proxy</string>
+    <string name="status_protected_with_socks5">Protected with SOCKS proxy.</string>
 
-    <string name="status_protected_with_http">protected with http proxy</string>
+    <string name="status_protected_with_http">Protected with HTTP proxy.</string>
 
-    <string name="status_protected_with_proxy">protected with proxy</string>
+    <string name="status_protected_with_proxy">Protected with proxy.</string>
 
-    <string name="status_strict" description="This is similar to dns_on, but the protection is being provided by the operating system, not by Rethink.  It is not color-highlighted.">
-    private dns is enabled
+    <string name="status_strict" description="Similar to dns_on, but the protection is provided by the operating system, not by Rethink. It is not color-highlighted.">
+    Private DNS on.
```

Comment:
For reduced complexity, all the "enable" is "turn on" or "on", and "disable" is "turn off" or "off", on the assumption that
nothing is ever disabled (subordinate) and turned on (primary function) in a way the user needs to know about.

Reply:
I can almost guarantee someone will translate that as "Private DNS on ___".